webapp/WEB-INF/templates/admin/plugins/forms/manage_control.html

```diff
@@ -1,72 +1,75 @@
 <@formBreadCrumb>
-	<li><@link href='jsp/admin/plugins/forms/ManageSteps.jsp?view=manageSteps&id_form=${step.idForm}' title='#i18n{forms.create_form.button.backForm}'>...</@link></li>
-	<li><@link href='jsp/admin/plugins/forms/ManageQuestions.jsp?view=manageQuestions&id_step=${step.id}' title='#i18n{forms.create_form.button.backStep}'>${step.title}</@link></li>
-	<li class="active">#i18n{forms.manage_controls.title.${control_type?lower_case}}</li></ul>
+<li><@link href='jsp/admin/plugins/forms/ManageSteps.jsp?view=manageSteps&id_form=${step.idForm}' title='#i18n{forms.create_form.button.backForm}'>...</@link></li>
+<li><@link href='jsp/admin/plugins/forms/ManageQuestions.jsp?view=manageQuestions&id_step=${step.id}' title='#i18n{forms.create_form.button.backStep}'>${step.title}</@link></li>
+<li class="active">#i18n{forms.manage_controls.title.${control_type?lower_case}}</li></ul>
 </@formBreadCrumb>
 
 <@modalBody>
-	<#if control_list?size gt 0>
-		<@table>
-			<thead>
-				<tr>
-					<@th xs=6 sm=5 md=7>#i18n{forms.modify_control.labelValidator}</@th>
-					<@th xs=5 sm=6 md=4>#i18n{forms.manageForm.columnActions}</@th>
-				</tr>
-			</thead>
-			<tbody>
-				<#list control_list as control>
-					<#assign validator = validatorManager.getValidator( control.validatorName ) />
-					<tr>
-						<@td sm=6>${validator.validatorDisplayName}</@td>
-						<@td sm=3>
-							<@aButton href='jsp/admin/plugins/forms/ManageControls.jsp?view=modifyControl&id_control=${control.id}' title='#i18n{forms.manage_questions.action.control}' hideTitle=['all'] buttonIcon='pencil' size='sm' />
-							<@aButton href='jsp/admin/plugins/forms/ManageControls.jsp?view=confirmRemoveControl&id_control=${control.id}' title='#i18n{forms.manage_controls.action.removeControl}' hideTitle=['all'] buttonIcon='trash' color='danger' size='sm' params=' target="_top"' />
-						</@td>
-					</tr>
-				</#list>
-			</tbody>
-		</@table>
-		<#--                
-			<#if control_type == 'VALIDATION'>
-				<@aButton href='jsp/admin/plugins/forms/ManageQuestions.jsp?view=manageQuestions&id_step=${step.id}' target='_top' title='#i18n{forms.form.returnPage}' size='' />
-			<#else>
-				<@aButton href='jsp/admin/plugins/forms/ManageTransitions.jsp?view=manageTransitions&id_step=${step.id}' target='_top' title='#i18n{forms.form.returnPage}' size='' />
-			</#if> 
-		-->
-	<#else>    
-		<@alert color='warning'>
-			<@icon style='exclamation-triangle' /> #i18n{forms.manage_transitions.labelNoControl}
-		</@alert>
-	</#if>    
-		<@messages errors=errors />
-		<@messages infos=infos />
+<#if control_list?size gt 0>
```

Comment:
add indentations in your code

Reply:
I have re-indenting those templates (modify_transition_control.html, manage_control) today in the commits named : lut_25506 : The transition to a step depending on a condition is no longer functional (re-indent)